css/mainresume.css

```diff
@@ -0,0 +1,46 @@
+.main-content {
+  position: absolute;
+  top: 8rem;
+  left: 10%;
+  width: 90%;
+}
+
+.content-card {
+  background-color: white;
+
+  /* max-width: 376px; */
+}
+
+.main-background {
+  background-color: #faefe9;
+  height: 2000px;
+}
+
+@media (min-width: 768px) {
+  .main-content {
+    margin-left: auto;
+    margin-right: auto;
+    top: 10rem;
+    /* left: 25%; */
+    /* max-width: 50%; */
+    min-width: 700px;
+  }
+  .main-background {
+    height: 2200px;
+  }
+  .content-card {
+    min-width: 700px;
+  }

```

Comment:
Using width when you are also using `col-...` classes on the same element is an issue. Choose one or the other 🙂